server/main.lua

```diff
@@ -11,26 +11,55 @@ local availableJobs = {
+    local firstname = Player.PlayerData.charinfo.firstname
+    local lastname = Player.PlayerData.charinfo.lastname
+    local gender = Player.PlayerData.charinfo.gender == 1 and 'Male' or 'Female'
+    local birthdate = Player.PlayerData.charinfo.birthdate
+    local nationality  = Player.PlayerData.charinfo.nationality
     for _, v in pairs(QBCore.Shared.StarterItems) do
-        local info = {}
-        if v.item == "id_card" then
-            info.citizenid = Player.PlayerData.citizenid
-            info.firstname = Player.PlayerData.charinfo.firstname
-            info.lastname = Player.PlayerData.charinfo.lastname
-            info.birthdate = Player.PlayerData.charinfo.birthdate
-            info.gender = Player.PlayerData.charinfo.gender
-            info.nationality = Player.PlayerData.charinfo.nationality
-        elseif v.item == "driver_license" then
-            info.firstname = Player.PlayerData.charinfo.firstname
-            info.lastname = Player.PlayerData.charinfo.lastname
-            info.birthdate = Player.PlayerData.charinfo.birthdate
-            info.type = "Class C Driver License"
+        if v.item == 'id_card' then
+            local metadata = {
+                type = string.format('%s %s', firstname, lastname),
+                description = string.format('CID: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', citizenid, birthdate, gender, nationality),
+                citizenid = citizenid,
+                firstname = firstname,
+                lastname = lastname,
+                gender = gender,
+                birthdate = birthdate,
+                nationality = nationality,
+            }
+            exports.ox_inventory:AddItem(source, v.item, v.amount, metadata)
+        elseif v.item == 'driver_license' then
+            local metadata = {
+                type = 'Class C Driver License',
+                description = string.format('CID: %s  \nFirst Name: %s  \nLast Name: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', citizenid, firstname, lastname, birthdate, gender, nationality),
+                citizenid = citizenid,
+                firstname = firstname,
+                lastname = lastname,
+                gender = gender,
+                birthdate = birthdate,
+                nationality = nationality,
+            }
+            exports.ox_inventory:AddItem(source, v.item, v.amount, metadata)
+        elseif v.item == "weaponlicense" then
+            local metadata = {
+                type = 'Weapon License',
+                description = string.format('CID: %s  \nFirst Name: %s  \nLast Name: %s  \nDOB: %s  \nGender: %s  \nNationality: %s ', citizenid, firstname, lastname, birthdate, gender, nationality),
+                citizenid = citizenid,
+                firstname = firstname,
+                lastname = lastname,
+                gender = gender,
+                birthdate = birthdate,
+                nationality = nationality,
```

Comment:
Why are we adding this metadata to the item? If we're capturing it in the description already, I'm trying to understand what use it has in the metadata as well?